cli/repl.rs

```diff
@@ -133,7 +133,7 @@ async fn post_message_and_poll(
         // A zero delay is long enough to yield the thread in order to prevent the loop from
         // running hot for messages that are taking longer to resolve like for example an
         // evaluation of top level await.
-        tokio::time::delay_for(tokio::time::Duration::from_millis(0)).await;
+        tokio::time::sleep(tokio::time::Duration::from_millis(0)).await;
```

Comment:
Unrelated to tokio 0.3 upgrade, but I think using `tokio::task::yield_now` would be better here.

Reply:
CC @caspervonb

Reply:
As far as I remember I tried that early and it spun hot at 100% but can take a look at it again.

Reply:
Yeah it'll turn hot.

![Screenshot 2020-10-20 at 8 47 03 PM](https://user-images.githubusercontent.com/157787/96587842-6f734b80-1315-11eb-8d8d-1730136ab9e8.png)